dev/com.ibm.ws.opentracing/src/com/ibm/ws/opentracing/OpentracingClientFilter.java

```diff
@@ -83,7 +83,9 @@ public void filter(ClientRequestContext clientRequestContext) throws IOException
 
         Tracer tracer = OpentracingTracerManager.getTracer();
         if (tracer == null) {
-            Tr.error(tc, "OPENTRACING_NO_TRACER_FOR_OUTBOUND_REQUEST");
```

Comment:
Given that this message is no longer used in code, it should be removed from nls file